### ISSUE TITLE
[ECP-9281] Multiple Shipment failed through API incase of the partial fulfillment with multi captures enabled

### ISSUE THREAD

**Issue by nareshanjuru** [Adyen/adyen-magento2#2653]
**Describe the bug**
In Magento 2.4.6.x after upgrading to Adyen Magento plugin 9.x we are facing challenges in creating multiple shipment API calls for partial fulfillment.

For example:
**To Reproduce**
Access the website, Add 2 products(P1 and P2) in the cart with qty 1 each and place order using Adyen credit card. Multiple captures enabled for the website in Adyen.
Through Magento shipment API and Invoice API:
1. create shipment for Product1 (P1), create invoice for the product1(P1) the order status and state changed to 'payment_review' in the vendor/adyen/module-payment/Observer/InvoiceObserver.php 
2. when we try to create shipment for the second product2 (P2) through API it gives the below error, till the capture webhook is processed and modify the order state and status

**{
    "message": "Shipment Document Validation Error(s):\nA shipment cannot be created when an order has a status of Payment Review",
    "errorCode": "MAGENTO_ERROR",
    "requestId": "15151c69-4646-4e4c-a21b-e486fcb3a9a9"
}**
It throws the error because Magento will not allow the orders with state 'payment_review' to do any further actions like shipment, Invoice and Cancel.

The files changed in the below commit:
https://github.com/Adyen/adyen-magento2/pull/2316/files#diff-deca6018cb1d4f03080dbdfd303a6afa6def6fcf516e90595838e2fe4c36d3cd

**Expected behavior**
Through API, it should allow to do the partial fulfillment of creating shipments, invoice without waiting for the webhook to process.

**Magento version**
2.4.6

**Plugin version**
9.x

**Desktop (please complete the following information):**
Postman API calls

**Comment by candemiralp** [comment]
Hello @nareshanjuru,

Thank you for creating this Github issue. Current approach of setting the state to `payment_review` is for preventing order fulfilments before successful captures. A capture request might fail and plugin can only be notified with the webhook.

However, we believe that there might be some improvements to allow consequent capture requests before processing the previous capture's webhook. I've created an internal investigation ticket for this enhancement.

Best regards,
Can

**Comment by raoulritter** [comment]
Hi @nareshanjuru,

Thank you for reporting this issue and providing detailed steps to reproduce. We've investigated the problem and considered various potential solutions. 

After careful consideration, we've decided to maintain the current implementation of setting the order state to `payment_review`. This approach, while causing some limitations, is crucial for preventing order fulfillment before successful captures are confirmed. This is especially important as capture requests might fail, and our plugin can only be notified of the outcome via webhooks.
